content/02.main-text.md

```diff
@@ -1,3 +1,21 @@
 ## Main text {.page_break_before}
 
-*Work in progress*
+### Introduction
+
+Open research – which includes sharing code, data, and manuscripts – benefits the researchers who engage in open practices [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+`TODO: more references needed`
+Here we advocate for the benefits of writing review articles openly, specifically planning, organizing, writing, and editing this type of manuscript collaboratively in an unrestricted public forum.
```

Comment:
I agree we can replace `advocate` with `report` or `describe`, but I do think we want to argue early on that there are benefits to doing it this way.  With such a tight word limit, I thought we could try to keep those benefits concentrated in the intro instead of returning to them in the conclusion.